src/App.js

```diff
@@ -186,19 +236,34 @@ class App extends Component {
         <BackTop />
         <Content>
           <div style={{ background: '#fff', padding: 24, minHeight: 280 }}>
+
+          <div style={{ textAlign: 'right', paddingTop: 16, paddingBottom: 16 }}>
```

Comment:
Should we be putting styles directly into the template, or would this be better done via CSS?

Reply:
Valid concern. I tend to prototype this way, while things are still in a state of flux. We can firm things up later.